lms/util/lti_launch.py

```diff
@@ -11,55 +11,46 @@ def get_application_instance(session, consumer_key):
     ).one()
 
 
-def default_get_secret(request, consumer_key):
+def get_secret(request, consumer_key):
     """Retrieve the lti secret given."""
```

Comment:
This docstring is a little vague; "given" maybe should be more explicit that it's coming from the `request`?